src/app/pages/common/entity/entity-tree-table/entity-tree-table.component.ts

```diff
@@ -116,7 +116,7 @@ export class EntityTreeTableComponent implements OnInit, AfterViewInit {
   }
 
   resolve(path: string, obj: any): string {
-    return path.split('.').reduce((prev, curr) => (prev ? prev[curr] : null), obj || self);
+    return path.split('.').reduce((prev, curr) => (prev ? prev[curr] : null), obj || self); // eslint-disable-line no-restricted-globals
```

Comment:
Original code looks like a mistake. Perhaps original author wanted `this` or `{}` instead of `self`.